README.md

```diff
@@ -6,7 +6,167 @@
 [![Hex.pm](https://img.shields.io/hexpm/dt/pluggy_elixir?style=flat-square)](https://hex.pm/packages/pluggy_elixir)
 [![codecov](https://img.shields.io/codecov/c/github/brainnco/pluggy_elixir?style=flat-square)](https://codecov.io/gh/brainnco/pluggy_elixir)
 
-**TODO: Add description**
+Welcome to Pluggy Elixir, an API client written in Elixir to access the Open Banking services provided by [Pluggy](http://pluggy.ai)
+
+## Use cases
+### Register
```

Comment:
I think `Installation` section must be the first section of README. WDYT about move this section down?